cmd/integration_link.go

```diff
@@ -174,7 +174,7 @@ List of available integrations:
 				awareOfSecurityRisks := c.Bool("aware-of-security-risks")
 
 				if awareOfSecurityRisks && !c.IsSet("allow-review-apps-from-forks") {
-					errorQuitWithHelpMessage(errors.New("flag --aware-of-security-risks is useless without allowing review apps from forks"), c, "integration-link-create")
+					errorQuitWithHelpMessage(errors.New("flag --aware-of-security-risks must be used in conjunction with --aware-of-security-risks"), c, "integration-link-create")
```

Comment:
Reading it out loud made me realize this sentence has no meaning: "flag F must be used in conjunction with F". I told you to update the text according to the comment I linked to, so... it's my fault.

I can't even write I didn't double check, since PM pinged me the suggested change.